pkg/rke/k8s_version_info.go

```diff
@@ -55,6 +55,10 @@ func loadRancherDefaultK8sVersions() map[string]string {
 		"2.8.1":        "v1.27.x",
 		"2.8.2":        "v1.27.x",
 		"2.8.3":        "v1.28.x",
+		"2.8.4":        "v1.28.x",
+		"2.8.5":        "v1.28.x",
+		"2.8.6":        "v1.28.x",
+		"2.8.7":        "v1.28.x",
```

Comment:
I think we don't need these as the branch is for 2.9 only, but it's not a breaking change. (I see the versions prior to 2.8.4 already existed, so maybe that's why it's getting maintained.)